client/src/App.jsx

```diff
@@ -1,24 +1,32 @@
-import React from 'react'
-import ReactDOM from 'react-dom'
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Header from './components/Header.jsx';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       random: '',
+      price: 'loading...',
     };
-//this bindings
   }
 
-//methods
+  componentDidMount() {
```

Comment:
I would personally build the fetch logic into a function outside of ComponentDidMount and then just call the func inside of CDM.

Reply:
Good idea